Makefile

```diff
@@ -31,6 +31,10 @@ clean:
 distclean: clean
 	find vendor/ -maxdepth 1 -mindepth 1 -not -name vendor.json -exec rm {} -rf \;
 	rm -f manifest/*.yaml
+	rm -f .glide.status
+
+checksync: .glide.status
+	test -d vendor || make sync
```

Comment:
glide does not need the vendor folder, why do you check for it? It will create it if it is not there.